core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -97,12 +97,12 @@ describe('Learner dashboard functionality', function() {
     // dashboard.
     await learnerDashboardPage.get();
     await learnerDashboardPage.navigateToSubscriptionsSection();
-    // The last user (collectionAdm) that learner subsribes to is placed first
+    // The last user (creatorName) that learner subsribes to is placed first
     // in the list.
-    await learnerDashboardPage.expectSubscriptionFirstNameToMatch('collect...');
-    // The first user (creatorName) that learner subscribes to is placed
+    await learnerDashboardPage.expectSubscriptionFirstNameToMatch('creator...');
+    // The first user (collectionAdm) that learner subscribes to is placed
     // last in the list.
-    await learnerDashboardPage.expectSubscriptionLastNameToMatch('creator...');
+    await learnerDashboardPage.expectSubscriptionLastNameToMatch('collect...');
```

Comment:
Why are we changing whether the user or email is displayed here? Shouldn't this PR just be a refactor, or are we including more substantive changes too?

Reply:
Yup, this PR is just for refactoring. But while refactoring I found a bug - #9797. So, I fixed that too.